recipes/anvio-minimal/meta.yaml

```diff
@@ -15,40 +15,34 @@ build:
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-  build:
-    - {{ compiler('c') }}
   host:
-    - python >=3,<3.7
-    - pip
-    - numpy
-    - cython
-    - gsl
+    - python ==3.6
```

Comment:
@ozcan all these strict pins are not recommended. Can you loosen them? Python 3.7 is probably also supported isn't it?

Reply:
Hi thank you for your review, I tried to answer version pinning here:
https://github.com/bioconda/bioconda-recipes/pull/17994#discussion_r333356134